src/transaction_utils.hpp

```diff
@@ -92,6 +92,9 @@ namespace sdk {
     amount add_tx_addressee(ga_session& session, const network_parameters& net_params, nlohmann::json& result,
         wally_tx_ptr& tx, nlohmann::json& addressee);
 
+    void set_tx_output_value(const network_parameters& net_params, wally_tx_ptr& tx, uint32_t index,
+        const std::string& asset_tag, amount::value_type satoshi);
```

Comment:
we don't use `asset_tag` any more since it was confused with `asset_id` in the past - this should be `asset_id`

Reply:
Changed to `asset_id`